src/codegen/function.rs

```diff
@@ -13,7 +13,7 @@ use super::translate_to_glib::TranslateToGlib;
 use writer::primitives::{format_block, tabs};
 use writer::ToCode;
 
-pub fn generate<W: Write>(w: &mut W, env: &Env, analysis: &analysis::functions::Info,
+pub fn generate(w: &mut Write, env: &Env, analysis: &analysis::functions::Info,
```

Comment:
I has problem with `save_to_file`.
Tried `W: Write, F: FnMut(&mut W) -> Result<()>` but then rust can't detect right type for closure yourself, so I decided change function to accept trait directly.